client/my-sites/guided-transfer/host-credentials-page/index.jsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Bluehost from './bluehost';
+import SiteGround from './siteground';
+import SectionHeader from 'components/section-header';
+
+class HostCredentialsPage extends Component {
+	constructor() {
+		super();
+		this.state = { fieldValues: {} };
```

Comment:
Two points here:
 1. We should use `getInitialState` instead of setting the initial state via `this.state = ...`
 2. We should probably use the redux store instead of state here and just avoid persisting if thats what we're going for.